src/pymmcore_widgets/hcs/_main_wizard_widget.py

```diff
@@ -0,0 +1,510 @@
+        layout.addWidget(self._fov_widget)
+        layout.addItem(QSpacerItem(0, 0, *EXPANDING))
+
+        self.setButtonText(QWizard.WizardButton.FinishButton, "Value")
+
+    def value(
+        self,
+    ) -> tuple[Plate | None, Center | RandomPoints | GridRowsColumns | None]:
+        """Return the list of FOVs."""
+        return self._fov_widget.value()
+
+    def setValue(
+        self, plate: Plate | None, mode: Center | RandomPoints | GridRowsColumns | None
+    ) -> None:
+        """Set the list of FOVs."""
+        self._fov_widget.setValue(plate, mode)
+
+
+class HCSWizard(QWizard):
+    """A wizard to setup an High Content experiment.
+
+    This widget can be used to select a plate, calibrate it, and then select the FOVs
+    to image in different modes (Center, RandomPoint or GridRowsColumns).
+
+    Parameters
+    ----------
+    parent : QWidget | None
+        The parent widget. By default, None.
+    mmcore : CMMCorePlus | None
+        The CMMCorePlus instance. By default, None.
+    plate_database_path : Path | str | None
+        The path to the plate database. By default, None.
+    """
+
+    valueChanged = Signal(object)
+
+    def __init__(
+        self,
+        parent: QWidget | None = None,
+        *,
+        mmcore: CMMCorePlus | None = None,
+        plate_database_path: Path | str | None = None,
+    ) -> None:
+        super().__init__(parent)
+        self._mmc = mmcore or CMMCorePlus.instance()
+
+        self.setWizardStyle(QWizard.WizardStyle.ModernStyle)
+        self.setWindowTitle("HCS Wizard")
+
+        # add custom button to save
+        self.setOption(QWizard.WizardOption.HaveCustomButton1, True)
+        _save_btn = self.button(QWizard.WizardButton.CustomButton1)
+        _save_btn.setText("Save")
+        self.setButton(QWizard.WizardButton.CustomButton1, _save_btn)
+        self.button(QWizard.WizardButton.CustomButton1).clicked.connect(self._save)
+        # add custom button to load
+        self.setOption(QWizard.WizardOption.HaveCustomButton2, True)
+        _load_btn = self.button(QWizard.WizardButton.CustomButton2)
+        _load_btn.setText("Load")
+        self.setButton(QWizard.WizardButton.CustomButton2, _load_btn)
+        self.button(QWizard.WizardButton.CustomButton2).clicked.connect(self._load)
+
+        # layout
+        layout = QVBoxLayout(self)
+        layout.setContentsMargins(0, 50, 0, 0)
+        layout.setAlignment(Qt.AlignmentFlag.AlignCenter)
+
+        # check if the default plate database exists, if not create it in the
+        # user data directory. If the database is provided, use it without storing it
+        # in the user data directory.
+        if not plate_database_path:
+            if not USER_PLATE_DATABASE_PATH.exists():
+                USER_PLATE_DATABASE_PATH.parent.mkdir(parents=True, exist_ok=True)
+            save_database(
+                load_database(DEFAULT_PLATE_DB_PATH), USER_PLATE_DATABASE_PATH
+            )
```

Comment:
i'm somewhat conflicted about the pattern of copying the default database to the user database.  perhaps consider saving only the user-provided wells to the user database, and preserving the pymmcore-provided database independently.  A nice object that can help here is a [ChainMap](https://docs.python.org/3/library/collections.html#collections.ChainMap).  Basically, it's a collection of dictionaries.  when you look up a key, if it's not in the first dict, it goes to the second dict, etc...  So, `load_database` could return a `ChainMap` with the default database at the end.  That would make it so that new releases of pymmcore-plus could extend the database, while also allowing a user to override any of theme (since earlier mappings in a chain map take precendence)